lib/ui/tabs/event_page.dart

```diff
@@ -85,7 +86,10 @@ class EventDetailPageState extends State<EventDetailPage> {
                     ),
                   ),
                   body: Center(
-                    child: CircularProgressIndicator(),
+                    child: SpinKitFadingCube(
+                      color: Color(0xffB54646),
+                      size: 50.0,
```

Comment:
The same also holds true for size, never hardcode size of something like this. Always create a separate file for your style constants.

But, for now, let's leave it hardcoded.